src/templates/homepage.html

```diff
@@ -20,7 +20,8 @@
 				<img class="chart-logo fade-in animation-delay__3" src="./img/chartjs-logo.svg"/>
 				<div class="fade-in fade-in animation-delay__5">
 					<h1 class="hero-title">Chart.js</h1>
-					<h2 class="hero-subtitle">Simple yet flexible JavaScript charting for designers &amp; developers</h2>
+					<h2 class="hero-subtitle">Most popular charting library in the world</h2>
```

Comment:
Even if this is true, I would not go this far. @etimberg thoughts?

Reply:
No I personally like the old one better, even if it's true I don't like showcasing with it, it feels wrong to me also if another lib takes over you need to update it again

Reply:
While I see nothing wrong using Chart.js' popularity for its own good, I won't insist on new wording. Open to revert it back.

However, I don't really get the "designers & developers" part. Do we really know/think that designers use Chart.js??? I would understand that about https://p5js.org—but Chart.js?

Reply:
I'm OK with it if we have some data to back it up. I did kind of like the old tag line which has been that way since the beginning (before I contributed to chartjs).

What about "Simple yet flexible JavaScript charting for the modern web"?

Reply:
Works for me! I've updated to exactly what you @etimberg suggested but added "library" to make it more clear what kind of product Chart.js is.
<img width="882" alt="Screen Shot 2022-12-14 at 19 43 26" src="https://user-images.githubusercontent.com/3852894/207641376-e03a0094-7ffc-44e1-81f8-17e9509e6d14.png">